src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "./utils/reset";
+@import "./utils/normalize";
```

Comment:
We can first mention reset and normalize, and after that variables.

Reply:
What to do in case if I need to use variable inside of normalize.scss? It simply won't be readable.